main/plugins/org.talend.sdk.component.studio-integration/src/main/java/org/talend/sdk/component/studio/util/TcompV0ConfigToFlatMapDeserializer.java

```diff
@@ -0,0 +1,48 @@
+package org.talend.sdk.component.studio.util;
+
+import java.io.StringReader;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonReader;
+import javax.json.JsonValue;
+
+public class TcompV0ConfigToFlatMapDeserializer {
+
+    public static Map<String,String> configToMap(String jsonConfig) {
+        JsonReader reader = Json.createReader(new StringReader(jsonConfig));
+        JsonObject configAsJsonObject = reader.readObject();
+
+        Map<String,String> result = new LinkedHashMap<>();
+
+        configAsJsonObject.keySet().stream().filter(key ->
+                configAsJsonObject.get(key) instanceof JsonObject).forEach(key -> {
+            JsonObject value = (JsonObject) configAsJsonObject.get(key);
+            findStoredValueInJson(key, result, value);
+        });
```

Comment:
Just suggestion for more readability
```suggestion
        configAsJsonObject.keySet().stream()
            .filter(key -> configAsJsonObject.get(key) instanceof JsonObject)
            .forEach(key -> {
                JsonObject value = (JsonObject) configAsJsonObject.get(key);
                findStoredValueInJson(key, result, value);
        });
```

Reply:
👍